bin/CMakeLists.txt

```diff
@@ -35,12 +35,27 @@ include(CMakeListsNrnMech)
 # =============================================================================
 nrn_configure_file(nrngui bin)
 nrn_configure_file(neurondemo bin)
-nrn_configure_file(nrnivmodl bin)
+nrn_configure_dest_src(nrnivmodl bin/tmp nrnivmodl bin)
```

Comment:
What is the reason for this?

Reply:
@nrnhines : we want to have `nrnivmodl` executable (`+x`) permission when its in `build` directory otherwise it's problem with `zsh` (IIRC).  Currently `+x` is added by CMake when we install with `make install`.